### ISSUE TITLE
Remove cache

### ISSUE THREAD

**Issue by Nuruddinjr** [gdg-x/hoverboard#64]
Enhancement: Remove cache in productionto firebase
When I deploy to firebase cache is not refreshed for users, so they have to clean their cache manually

**Comment by ozasadnyy** [comment]
Currently, you can use `networkFirst` SW strategy instead of `fastest` in [index.html](https://github.com/gdg-x/hoverboard/blob/master/app/index.html#L16)
```
  <platinum-sw-cache default-cache-strategy="networkFirst"
                     cache-config-file="/cache-config.json">
```

**Comment by limhenry** [comment]
this could fix the problem right: show toast if new version is available

```
reg.onupdatefound = function() {
    var installingWorker = reg.installing;
    installingWorker.onstatechange = function() {
        switch (installingWorker.state) {
            case 'installed':
                if (navigator.serviceWorker.controller) {
                   //show toast if new version is available
                }
                break;
            case 'redundant':
                break;
        }
    }
}
```

_but the problem is you can't use pt-sw (maybe?)_

**Comment by ozasadnyy** [comment]
@limhenry awesome! it would be implemented in a new version.
Stay tuned for the updates.

**Comment by ozasadnyy** [comment]
Fixed in #100 . 
Now when new service worker appears, user is been asking to reload a page.
